### DISCUSSION TITLE
System Requirements & Real World Experiences with OpenRVDAS on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by cromsos** [OceanDataTools/openrvdas#396] [General]
A colleague asked me yesterday if OpenRVDAS can run on a Raspberry Pi, and if so which Pi to order?  They told me that their intent was to run about 6 data loggers with minimal (prefix type) transformations storing data to local removable media.

I see that the installation script supports Raspbian, so I answered 'yes' to the first question but would like to give them some advice on performance - based on user experience.  Maybe to simplify the ask, can a Pi run the full OpenRVDAS install including the django GUI?

Thanks

**Comment by davidpablocohn** [comment]
Beyond number of loggers, I'd also consider the data rate - six gyros? Without blinking. Six winches at 20 Hz? I might want to do a benchmark or two. But for typical 1-5 Hz loggers, a Raspberry Pi should have no problems. The machine I installed at Palmer Station was a Pi 4B, running both full OpenRVDAS and the InfluxDB/Grafana suite without noticeably loading up (see: https://davidpablocohn.com/2023/04/03/getting-stuff-done/ for a couple of pics). Nowadays, there's apparently a Pi 5 that's even more powerful.

**Reply by cromsos** [reply]
Thanks!  That's what I was looking for.  -Chris
